vagrant/Vagrantfile

```diff
@@ -66,6 +66,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
               if [ $? = 0 ]; then 
                 break
               fi
+              echo "Retry to start PM2 Daemon"
+              pm2 status  # any PM2 command will start PM2 Daemon. and 'pm2 status' is a blocking command.
```

Comment:
don't see which parts try for 30 times. and what does it mean for 'any PM2 command will start PM2 Daemon'?